ember/5/convert-module-for-to-setup-test/test.ts

```diff
@@ -0,0 +1,2176 @@
+import { FileInfo } from 'jscodeshift';
+import assert from 'node:assert';
+import transform from './index.js';
+import { buildApi } from '../../../utilities.js';
+
+describe('ember 5 convert-module-for-to-setup-test', function () {
```

Comment:
the tests here are likely licensed as well, we need to provide a license here as well.